src/login/logind-session-device.c

```diff
@@ -104,6 +105,20 @@ static void sd_eviocrevoke(int fd) {
         }
 }
 
+static void sd_hidiocrevoke(int fd) {
+        static bool warned = false;
+
+        assert(fd >= 0);
+
+        if (!warned && ioctl(fd, HIDIOCREVOKE, NULL) < 0) {
+                if (errno == EINVAL) {
+                        log_warning_errno(errno, "Kernel does not support hidraw-revocation: %m");
```

Comment:
we usually want that the string of warning log messages says what we are actually doing now given that something failed but in a non-fatal way. This often just means you should end the sentence in `…, ignoring: %m` (like you already do below). But sometimes something more specific would be better, maybe here: `…, proceeding without revoking device access`.

Reply:
done, fixed as requested (for both messages)